utilities/manual/usermanual_en.html

```diff
@@ -619,7 +619,7 @@ <h2 id="SECTION33">4.6 Personal macros</h2>
 The needed functionality can be selected by checkbox.<br>
 The "abbreviation" is a pseudo-command for the latex completer. If the pseudo-command is completed, the macro will be inserted instead. Note that the pseudo-command needs to start with a backslash ("\").<br>
 "Trigger" is a regular expression which triggers the inclusion of the macro: When the last written characters match this expression, they are removed and the macro is inserted/executed. (see <a href="#sectionTriggers">below</a> for more details).<br>
-Some macros can be directly downloaded from an internet repository. The dialog is started with the button "Browse". For easier data exchange, macros can be im- and exported to a file. If you want to add a macro of your own to that repository, you can hand it in as a feature request on <a href="https://github.com/texstudio-org/texstudio/issues">Github</a>. <br>
+Some macros can be directly downloaded from an internet repository. The dialog is started with the button "Browse". For easier data exchange, macros can be im- and exported to a file (one by one). If you want to add a macro of your own to that repository, you can hand it in as a feature request on <a href="https://github.com/texstudio-org/texstudio/issues">Github</a>. <br>
```

Comment:
The previous wording strongly suggests that all macros can be imported/exported to a file at once. But this is not true or the words are very missleading. From data exchange I expected it would be true, but I was disappointed. As long as issue #1890 is not implemented you can only export one macro per file. I think this is worth clarifying.

Reply:
It generally describes that macros can be saved.
If need be, I can accept "can be saved individually"

Reply:
I will remove it